ui_curses.c

```diff
@@ -2326,14 +2330,18 @@ static void init_curses(void)
 	update_mouse();
 }
 
-static void notify_init(void)
+static int notify_init(void)
 {
 	int fildes[2];
-	pipe(fildes);
+	if (pipe(fildes) == -1) {
+		error_msg("pipe failed: %s\n", strerror(errno));
```

Comment:
`error_msg` here makes no sense — no one will see it, because cmus will exit immediately. Please use `d_print` here, too.